src/main/java/com/linecorp/armeria/server/http/tomcat/TomcatService.java

```diff
@@ -73,6 +76,10 @@ public static TomcatService forFileSystem(Path docBase) {
         return TomcatServiceBuilder.forFileSystem(docBase).build();
     }
 
+    public static HttpService forConnector(String hostname, Connector connector) {
```

Comment:
1) IIUC this method returns `HttpService` rather than `TomcatService` because a handler created from a `Connector` can't have `TomcatServiceConfig`, right? It feels awkward that only this method has a different return type.

How about adding `connector()` method to `TomcatService` and throwing an `UnsupportedOperationException` in `config()` when constructed via this method?

2) Please add Javadoc.